.evergreen/install-node.sh

```diff
@@ -94,6 +96,10 @@ if [[ "$file_extension" = "zip" ]]; then
     echo "Node.js already installed!"
   else
     "$SCRIPT_DIR/retry-with-backoff.sh" curl "${CURL_FLAGS[@]}" "${node_download_url}" --output "$node_archive_path"
+    "$SCRIPT_DIR/retry-with-backoff.sh" curl "${CURL_FLAGS[@]}" "${node_shasum_url}" --output "$node_shasum_path"
+    pushd $NODE_ARTIFACTS_PATH
+    sha256sum -c SHASUMS256.txt --ignore-missing
+    popd
```

Comment:
```suggestion
    (cd "$NODE_ARTIFACTS_PATH" && sha256sum -c SHASUMS256.txt --ignore-missing)
```
Suggestion: a bit simpler than pushd/popd?